src/content/get-started/flutter-for/web-devs.md

```diff
@@ -745,7 +745,7 @@ with [`BoxShape` enum][] for this purpose.
 }
 ```
 
-```dart highlightLines=10,13-14,18
+```dart highlightLines=11,14-15
```

Comment:
Do you think we should include `19` in here to highlight the `textAlign: TextAlign.center,` that corresponds to the highlighted `text-align: center;` in the CSS? Or another option would be to stop highlighting the `text-align: center;` in the CSS block.

````suggestion
```dart highlightLines=11,14-15,19
````